lib/EditableList/ItemEdit.js

```diff
@@ -50,7 +50,7 @@ const ItemEdit = ({
       if (Object.hasOwnProperty.call(fieldComponents, name)) {
         return (
           <div key={fieldKey} style={fieldStyle}>
-            { fieldComponents[name]({ fieldProps, fieldIndex, name, field, rowIndex }) }
+            { fieldComponents[name]({ fieldProps, fieldIndex, name, field, rowIndex, error }) }
```

Comment:
Is there a related PR for this change? This passed object doesn't match how it's passed below
It should be added to `fieldProps`
```
error: error?.fieldErrors?.[name]
```